packages/react/package.json

```diff
@@ -35,12 +35,13 @@
     "build:ts": "tsup --minify --keep-names",
     "build:catalog": "ts-node scripts/generatePrimitivesCatalog.ts",
     "dev": "tsup --watch src",
+    "eslint:primitives": "npx eslint ./src/primitives --ext .js,.ts,.tsx",
```

Comment:
Why do we want to lint only primitives? I think it would be an overall benefit to run rules of hook (and other lint rules) and all the components. Thoughts?

Reply:
Yeah, I'm starting with the primitives since I knew how to fix those errors. Will follow up with a draft PR to run eslint on all the files so I can get help on some of the Authenticator lint errors.